Kudu.Core.Test/Jobs/ScheduleFacts.cs

```diff
@@ -31,12 +31,20 @@ public void ScheduleProvidesNextIntervalCorrectly(string cronExpression, string
 
             var triggeredJobSchedulerLoggerMock = new Mock<TriggeredJobSchedulerLogger>(String.Empty, mockEnvironment.Object, null);
             var schedule = Schedule.BuildSchedule(cronExpression, triggeredJobSchedulerLoggerMock.Object);
-            schedule.SetDateTimeProvider(new TestDateTimeNowProvider(DateTime.Parse("00:00:00 1/1/2015", CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal)));
+            var now = DateTime.Parse( "00:00:00 1/1/2015" , CultureInfo.InvariantCulture , DateTimeStyles.AssumeLocal );
+            schedule.SetDateTimeProvider( new TestDateTimeNowProvider( now ) );
 
             DateTime lastSchedule = DateTime.Parse(lastScheduleStr, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal);
             TimeSpan actualNextSchedule = schedule.GetNextInterval(lastSchedule, ignoreMissed);
 
             TimeSpan expectedNextInterval = TimeSpan.Parse(expectedNextIntervalStr);
+
+            var nowTimeOffset = new DateTimeOffset( now ).Offset;
```

Comment:
Since you do this same calculation below in the other test, please factor this out into a helper:

```csharp
TimeSpan GetTimeZoneAdjustedInterval(TimeSpan interval, DateTime now)
```